content/developer/release-and-packaging/determine-how-you-will-distribute-your-application.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: practice
+weight: 3
+title: Determine how you will distribute your application
+what: >
+  Choose how users will install your application/library.
+why: >
+  This will determine the tooling you need to use to package your application. Ideally this will be where your community goes to get its tools.
+when: Mid-Development
+importance: Medium
+implementations:
+  - desc: Conda
+  - desc: PyPI
+  - desc: Docker
+recommendation: conda via conda-forge
+see_also:
+  - "[Python Packaging Tutorial](https://packaging.python.org/en/latest/tutorials/packaging-projects/)"
+  - "[Grayskull - Conda package generator](https://github.com/conda/grayskull)"
+  - "[Conda-Forge Package Maintainer Guide](https://conda-forge.org/docs/maintainer/adding_pkgs/#creating-recipes)"
+---
```

Comment:
Again, I think it would be good to mention that the focus on conda is a little bit of a peculiarity of our ecosystem, and that more generally pip is the recommended method.

It's also probably worth mentioning that these aren't mutually exclusive, and if you can, you should probably distribute along all 3 channels.